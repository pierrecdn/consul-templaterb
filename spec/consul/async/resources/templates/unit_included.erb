<%= File.basename(template_info['source']) %>, root=<%= File.basename(template_info['source_root']) %>
was_rendered_once: <%= template_info['was_rendered_once'] %>
destination: <%= File.basename(template_info['destination']) %>