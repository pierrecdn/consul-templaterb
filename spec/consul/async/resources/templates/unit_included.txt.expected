unit_included.erb, root=unit_included.erb
was_rendered_once: true
destination: unit_included.txt